contracts/GenArt721CoreV3.sol

```diff
@@ -114,15 +112,19 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
      * @param _tokenName Name of token.
      * @param _tokenSymbol Token symbol.
      * @param _randomizerContract Randomizer contract.
+     * @param _adminACLContract Address of admin access control contract, to be
+     * set as contract owner.
      */
     constructor(
         string memory _tokenName,
         string memory _tokenSymbol,
-        address _randomizerContract
+        address _randomizerContract,
+        address _adminACLContract
     ) ERC721(_tokenName, _tokenSymbol) {
-        isWhitelisted[msg.sender] = true;
         artblocksAddress = payable(msg.sender);
         randomizerContract = IRandomizer(_randomizerContract);
+        // set AdminACL management contract as owner
+        _transferOwnership(_adminACLContract);
```

Comment:
I feel like we may want to write this explicitly onto a variable as well to expose it directly for introspection purposes–though maybe not worth doing, unsure.

Reply:
agreed - updated to override and wrap `_transferOwnership` to also update public variable `IAdminACLV0 public adminACLContract` whenever ownership is transferred.

Reply:
Fantastic–TYVM! :)